assets/js/screens/Collection/List.jsx

```diff
@@ -20,13 +28,40 @@ const ScreensCollectionList = () => {
     }
   }, [data]);
 
+  const [deleteCollection] = useMutation(DELETE_COLLECTION, {
+    onCompleted({ deleteCollection }) {
+      addToast(`Collection ${deleteCollection.name} deleted successfully`, {
+        appearance: "success",
+        autoDismiss: true
+      });
+    },
+    refetchQueries(mutationResult) {
+      return [{ query: GET_COLLECTIONS }];
+    }
+  });
+
   if (loading) {
     return <Loading />;
   }
   if (error) {
     return <Error error={error} />;
   }
 
+  const onOpenModal = (collectionId, collectionName) => {
+    setActiveCollection({id: collectionId, name: collectionName});
+    setModalOpen(true);
+  };
+
+  const onCloseModal = () => {
+    setActiveCollection({});
```

Comment:
So, here's a weird (kinda) thing with JavaScript and empty objects.  `{ foo: "bar" } === true` and `{} === true`.  An empty object will evaluate to `true`.   Sometimes we do checks on variables like `if(someVar) { ... }`

So, let's change line 56 to be `setActiveCollection();` or if that doesn't seem to work, `setActiveCollection(null);`

Reply:
I'm just verifying something and realized the code fails if i setActiveCollection() to anything other than an empty object since i read this object ID on 
```
<UIModalDelete
        isOpen={modalOpen}
        handleClose={onCloseModal}
        handleConfirm={handleDeleteClick}
        thingToDeleteLabel={`Collection ${activeCollection.name}`}
      />
```

Reply:
We discussed this offline and how to account for it with a ternary condition.